user_sync/connector/directory_csv.py

```diff
@@ -37,12 +37,16 @@ def connector_initialize(options):
     state = CSVDirectoryConnector(options)
     return state
 
-def connector_load_users_and_groups(state, groups):
+def connector_load_users_and_groups(state, groups, extended_attributes):
     '''
     :type state: CSVDirectoryConnector
     :type groups: list(str)
+    :type extended_attributes: list(str)
     :rtype (bool, iterable(dict))
     '''
+    if (extended_attributes is not None):
+        self.logger.warning("CSV directory connector doesn't support extended_attributes; ignored")
```

Comment:
Really?  Seems like it would be really really easy for the CSV connector to support extended attributes, since you could just add them as column headers.  And this seems like it would be a really, really handy way to test extended logic on just a few people...